src/components/include/application_manager/application_manager.h

```diff
@@ -87,7 +89,10 @@ class RPCService;
 namespace rpc_handler {
 class RPCHandler;
 }
-
+namespace request_controller {
+class RequestTimeoutHandler;
+class RequestController;
+}  // namespace request_controller
```

Comment:
```suggestion
} // namespace request_controller
```

Reply:
@ShobhitAd fixed in https://github.com/smartdevicelink/sdl_core/pull/3726/commits/57e4accc52957086f89470059f8fa07d9ee5dd7c